docs/examples/swift-api-walkthrough/api-walkthrough/ViewController.swift

```diff
@@ -175,10 +178,108 @@ class ViewController: UIViewController {
+        joinQuery()
+        groupByQuery()
+    }
+
+    func loadTravelSample() {
+        let dbPath = Bundle.main.path(forResource: "mini-travel-sample", ofType: "cblite2")!
+        let fileManager = FileManager.default
+        do {
+            let destinationPath = fileManager.urls(for: .applicationSupportDirectory, in: .userDomainMask)[0].appendingPathComponent("CouchbaseLite").appendingPathComponent("mini-travel-sample.cblite2").path
+            try fileManager.copyItem(atPath: dbPath, toPath: destinationPath)
+        } catch let error {
+            print(error.localizedDescription)
+        }
+    }
+    
+    func selectOverviewQuery() {
+        let query = Query.select(
+                SelectResult.expression(Expression.property("airportname")),
+                SelectResult.expression(Expression.property("city"))
+            )
+            .from(DataSource.database(database!))
+            .where(
+                Expression.property("type").equalTo("airport")
+                .and(Expression.property("tz").equalTo("Europe/Paris"))
+                .and(Expression.property("geo.alt").greaterThanOrEqualTo(300))
+            )
+        do {
+            for row in try query.run() {
+                print("\(row.toDictionary()))")
+            }
+        } catch {
+            
+        }
+    }
+    
+    func joinQuery() {
+        let query = Query.select(
+            SelectResult.expression(Expression.property("name").from("airline")),
+            SelectResult.expression(Expression.property("callsign").from("airline")),
+            SelectResult.expression(Expression.property("destinationairport").from("route")),
+            SelectResult.expression(Expression.property("stops").from("route")),
+            SelectResult.expression(Expression.property("airline").from("route"))
+        )
+        .from(
+            DataSource.database(database!).as("airline")
+        )
+        .join(
+            Join.join(DataSource.database(database!).as("route"))
+            .on(
+                Expression.meta().id.from("airline")
+                .equalTo(Expression.property("airlineid").from("route"))
+            )
+        )
+        .where(
+            Expression.property("type").from("route").equalTo("route")
+            .and(Expression.property("type").from("airline").equalTo("airline"))
+            .and(Expression.property("sourceairport").from("route").equalTo("RIX"))
+        )
+        do {
+            for row in try query.run() {
+                print("\(row.toDictionary()))")
+            }
+        } catch {
+            
+        }
+    }
+    
+    // missing the count
+    // see https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html
+    func groupByQuery() {
+        let query = Query.select(
+                SelectResult.expression(Expression.property("country")),
+                SelectResult.expression(Expression.property("tz"))
+            )
```

Comment:
not sure if DB14 supports `count` in the SELECT clause? N1QL equivalent query https://developer.couchbase.com/documentation/server/4.6/n1ql/n1ql-language-reference/selectintro.html#topic_xwp_r3k_dx__group-sort-limit